macros/SVGRef.ejs

```diff
@@ -66,7 +66,7 @@ var resultAPI = [];
             <li><%- template("domxref", [resultAPI[aTitle]]) %></li>
   <%    } 
     } %>
-   <li><a href="<%-s_svg_href%>"><%-s_svg_ref_title%></a><ol>
+   <li><details><summary><%-s_svg_ref_title%></summary><ol>
```

Comment:
Here's the opening ``<details>`` tag, so I'd expect a closing ``</details>`` eventually.